private/shell/utils/providers.ts

```diff
@@ -0,0 +1,102 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+
+import { logSpinner } from '.';
+import { outputDirectory } from '../constants';
+
+/**
+ * Get the name of the provider based on the environment variables.
+ *
+ * @returns A string representing the provider name.
+ */
+export const getProvider = (): typeof Bun.env.__BLADE_PROVIDER => {
+  if (Bun.env['CF_PAGES']) return 'cloudflare';
+  if (Bun.env['VERCEL'] === '1') return 'vercel';
```

Comment:
```suggestion
  if (Bun.env['VERCEL']) return 'vercel';
```

I think this is enough, right? Like on Cloudflare.